README.md

```diff
@@ -497,10 +964,17 @@
     // declaration to the top of the scope,
     // which means our example could be rewritten as:
     function example() {
-      var declaredButNotAssigned;
+      let declaredButNotAssigned;
       console.log(declaredButNotAssigned); // => undefined
       declaredButNotAssigned = true;
     }
+
+    // using const and let
+    function example() {
+      console.log(declaredButNotAssigned); // => throws a ReferenceError
+      console.log(typeof declaredButNotAssigned); // => throws a ReferenceError
+      const declaredButNotAssigned = true;
+    }
     ```
 
   - Anonymous function expressions hoist their variable name, but not the function assignment.
```

Comment:
Nice find. This is meant to be `var` and is a victim of some serial regex replace.